sal_interfaces/TunableLaser/TunableLaser_Events.xml

```diff
@@ -43,7 +43,7 @@
             <EFDB_Name>heartbeat</EFDB_Name>
             <Description>Beats the drum.</Description>
```

Comment:
?

Reply:
Copy and paste from Russell's heartbeat event.